src/core/mavlink_commands.h

```diff
@@ -12,10 +12,97 @@ namespace mavsdk {
 
 class SystemImpl;
 
-class MAVLinkCommands {
+class MavlinkCommandReceiver {
 public:
-    explicit MAVLinkCommands(SystemImpl& parent);
-    ~MAVLinkCommands();
+    explicit MavlinkCommandReceiver(SystemImpl& parent);
```

Comment:
```suggestion
    explicit MavlinkCommandReceiver(SystemImpl& system_impl);
```

This `parent` is my mistake but we should get away from it. It's wrong.